scripts/update/Updater.php

```diff
@@ -49,5 +51,17 @@ public function update($initialVersion)
             );
             $this->setVersion('0.5.0');
         }
+
+        if ($this->isVersion('0.5.0')){
+            $simpleKeyProvider = new SimpleKeyProviderService([]);
```

Comment:
Not needed if FileKeyProviderService is registered after

Reply:
no necessary